Too many queries request sent

With the following url, the application loads with only one leaf activated in the tree. However when clicking on the map to do a query, 4 different requests are sent to the server.
Here are the different type names that are requested:
 - feature:osm_time,
 - feature:sustenance,
 - feature: (x2)

Only the first one is valid. The other 3 are not.

https://camptocamp.github.io/ngeo/master/examples/contribs/gmf/apps/desktop/?baselayer_ref=map&lang=en&map_x=647472&map_y=178630&map_zoom=0&tree_group_layers_OSM%20functions=osm_time&tree_groups=OSM%20functions

Additional note:
There's a assertion error in the console because one of the request (the "feature:" one) responds with `ows:ExceptionReport`
```
AssertionError: Assertion failed: localName should be FeatureCollection
```

--- a/contribs/gmf/src/services/querymanager.js
+++ b/contribs/gmf/src/services/querymanager.js
@@ -152,9 +152,7 @@ gmf.QueryManager.prototype.createSources_ = function(node, ogcServers) {
           }
         }, this);
         layers = childLayerNames.join(',');
-        if (node.type === 'WMS' && childLayerNames.length == 1) {
-          validateLayerParams = true;
-        }
+        validateLayerParams = node.type === 'WMS';
       }
 
       var source = {

--- a/externs/ngeox.js
+++ b/externs/ngeox.js
@@ -520,15 +520,12 @@ ngeox.QuerySource.prototype.url;
  * to a layer that also has an ol3 WMS source object, then the latter may
  * contain more than one layer name within the LAYERS param. In that case,
  * this `validateLayerParams` property, when enabled, will make the query
- * service check if the layer name within its LAYERS params is currently inside
- * the layer source LAYERS params. If it's not there, then the source should
- * not be queried.
+ * service check if the layer name(s) within its LAYERS params is currently
+ * inside the layer source LAYERS params. If it's not there, then the source
+ * should not be queried.
  *
  * When setting this option, you must not set the wmsSource or layer if
  * it has an inner ol3 wms source object.
- *
- * Also, when using this option, your config `params` must also only have
- * one layer name set in the LAYERS property.
  * @type {boolean|undefined}
  */
 ngeox.QuerySource.prototype.validateLayerParams;

--- a/src/services/query.js
+++ b/src/services/query.js
@@ -423,10 +423,12 @@ ngeo.Query.prototype.getQueryableSources_ = function(map, wfsOnly) {
             'validateLayerParams option.'
         );
         var layerLayers = layerSource.getParams()['LAYERS'].split(',');
-        var cfgLayer = item.source.wmsSource.getParams()['LAYERS'];
-        goog.asserts.assert(cfgLayer.indexOf(',') === -1,
-            'The LAYERS param contains more than one item');
-        if (layerLayers.indexOf(cfgLayer) === -1) {
+        var cfgLayer = item.source.wmsSource.getParams()['LAYERS'].split(',');
+
+        var every = cfgLayer.every(function(layer) {
+          return layerLayers.indexOf(layer) != -1;
+        });
+        if (!every) {
           continue;
         }
       }

